### DISCUSSION TITLE
How to run it?

### DISCUSSION THREAD

**Discussion by PashaKagan** [openblack/openblack#294] [Q&A] [answered]
Sorry for the noob question.
I'm on Windows, finished building, how do I actually run the game?

**Comment by captnfab** [comment, marked as answer]
For reference, since the question has been posted a long time ago:
1. Build the game <https://github.com/openblack/openblack#building>
2. Extract assets from original game <https://github.com/openblack/openblack#extracting-asset-files-from-original-game>
3. Run the game with the following parameter: `-g Path/To/Assets`

In case it doesn't work, you may need to specify other parameters. Launch game in terminal with `-h` parameter to get a list of available options.
